src/components/Contact.tsx

```diff
@@ -19,6 +19,13 @@ const ContactInfo = styled.p<ContactPageProps>`
   text-transform: uppercase;
 `;
 
+const EmailLink = styled.a<ContactPageProps>`
+  &:hover {
+    text-decoration: underline;
+    transition-duration: 0.5s;
```

Comment:
this `duration` doesnt have an effect since theres no formal "transition" set.

Reply:
yep true